code/modules/mob/living/carbon/xenomorph/XenoOverwatch.dm

```diff
@@ -89,7 +89,11 @@
 			to_chat(src, SPAN_XENOWARNING("We can't watch ourselves!"))
 			return
 
-		if(targetXeno.interference)
+		if(HAS_TRAIT(src, TRAIT_HIVEMIND_INTERFERENCE))
+			to_chat(src, SPAN_XENOWARNING("Our psychic connection is cut off!"))
+			return
+
+		if(HAS_TRAIT(targetXeno, TRAIT_HIVEMIND_INTERFERENCE))
 			to_chat(src, SPAN_XENOWARNING("Our sister's psychic connection is cut off!"))
 			return
```

Comment:
These do successfully prevent new overwatch attempts for both sides, but doesn't end any existing. I think we should do the same two checks in `/mob/living/carbon/xenomorph/proc/handle_overwatch()` (as in check if we, or who we're observing gets interference - only for the case who we're observing gets interference should we have a `to_chat(src, SPAN_XENOWARNING("Our sister's psychic connection is cut off!"))`